src/IssueViz/Models/AnalysisIssueLocationVisualization.cs

```diff
@@ -45,21 +47,33 @@ public interface IAnalysisIssueLocationVisualization
         SnapshotSpan? Span { get; set; }
     }
 
-    public class AnalysisIssueLocationVisualization : IAnalysisIssueLocationVisualization, INotifyPropertyChanged
+    public class AnalysisIssueLocationVisualization : IAnalysisIssueLocationVisualization
     {
         private bool isNavigable;
+        private string filePath;
 
         public AnalysisIssueLocationVisualization(int stepNumber, IAnalysisIssueLocation location)
         {
             StepNumber = stepNumber;
             Location = location;
             IsNavigable = true;
+            FilePath = location.FilePath;
         }
 
         public int StepNumber { get; }
 
         public IAnalysisIssueLocation Location { get; }
 
+        public string FilePath
```

Comment:
New property: settable FilePath that the tagger could update.

Reply:
It's not obvious why the viz type has a property of the same name as the `Location` it wraps.
It would help if the property was called something different e.g. `CurrentFilePath`, with a doc comment on the interface explaining why it can be different from the underlying `Location.FilePath`.